packages/clarity-js/src/performance/observer.ts

```diff
@@ -48,28 +45,34 @@ function handle(entries: PerformanceObserverEntryList): void {
 
 function process(entries: PerformanceEntryList, offset: number): void {
     if (entries && entries.length > offset) {
+        let visible = "visibilityState" in document ? document.visibilityState === "visible" : true;
         for (let i = offset; i < entries.length; i++) {
             let entry = entries[i];
             switch (entry.entryType) {
                 case "navigation":
                     navigation.compute(entry as PerformanceNavigationTiming);
                     break;
                 case "resource":
-                    network.compute(entry as PerformanceResourceTiming);
-                    break;
-                case "paint":
-                    paint.compute(entry);
+                    dimension.log(Dimension.NetworkHosts, host(entry.name));
                     break;
                 case "longtask":
-                    longtask.compute(entry as LongTaskEntry);
+                    metric.count(Metric.LongTaskCount);
+                    break;
+                case "first-input":
+                    if (visible) { metric.count(Metric.FirstInputDelay, entry["processingStart"] - entry.startTime); }
```

Comment:
>count [](start = 42, length = 5)

this feel weird - should this be max or sum? (i know it doesn't make a difference since we would only get this once per page but count seems like it is a incremental thing)